.github/workflows/main.yaml

```diff
@@ -0,0 +1,307 @@
+name: Build and Deploy Image
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - workflow
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  GITHUB_IMAGE_REPO: ghcr.io/bcgov/dts-endorser-service/
+  OPENSHIFT_IMAGE_REPO: image-registry.apps.silver.devops.gov.bc.ca/4a9599-tools/
+  APP_NAMES: aries-endorser-agent,aries-endorser-db,aries-endorser-backup,aries-endorser-proxy,aries-endorser-api
+
+jobs:
+  build:
+    if: (github.repository == 'bcgov/dts-endorser-service') || (github.event_name == 'workflow_dispatch')
+    name: Build Image
+    permissions:
+      packages: write
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - service: aries-endorser-agent
+            GIT_REPO_URL: hyperledger/aries-endorser-service 
+            GIT_REF: ""
+            DOCKER_FILE_PATH: Dockerfile.acapy # The docker path, file, is the relative path to the docker file from the root of the repo.
+            SOURCE_CONTEXT_DIR: docker/acapy   # The context dir, context, sets the context for the build. i.e. where the build will source files from 
+            SOURCE_IMAGE_REGISTRY: ""
+            SOURCE_IMAGE_NAME: ""
+            SOURCE_IMAGE_TAG: ""
+            REGISTRY_USERNAME_SECRET_NAME: ARTIFACTORY_USERNAME 
+            REGISTRY_PASSWORD_SECRET_NAME: ARTIFACTORY_PASSWORD
+          - service: aries-endorser-db
+            GIT_REPO_URL: hyperledger/aries-endorser-service
+            GIT_REF: ""
+            PATH:  docker/wallet/config
+            SOURCE_IMAGE_REGISTRY: "quay.io/"  
+            SOURCE_IMAGE_NAME: "fedora/postgresql-13"
+            SOURCE_IMAGE_TAG: "13"
+          - service: aries-endorser-backup
+            GIT_REPO_URL: BCDevOps/backup-container
+            GIT_REF: ""
```

Comment:
The Backup container should be built off a release ref.  Refer to the OCP build param file.